spec/features/user_integration_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.feature 'User Integration', type: :feature do
+  let!(:user) { create(:user) }
+
+  scenario 'User index page' do
+    visit users_path
+
+    expect(page).to have_content(user.name)
+    expect(page).to have_selector('img.img-thumbnail')
+    expect(page).to have_content("Number of posts: #{user.posts_counter}")
+
+    click_link user.name
+    expect(current_path).to eq(user_path(user))
+  end
+
+  scenario 'User show page' do
+    visit user_path(user)
+
+    expect(page).to have_selector('img.img-thumbnail')
+    expect(page).to have_content(user.name)
+    expect(page).to have_content("Number of posts: #{user.posts_counter}")
+    expect(page).to have_content(user.bio)
+
+    if user.posts.any?
+      user.posts.limit(3).each do |post|
+        expect(page).to have_content(post.title)
+      end
+
+      expect(page).to have_link('See all Posts', href: user_posts_path(user))
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=post%27s%20show%20page.-,When%20I%20click%20to%20see%20all%20posts%2C%20it%20redirects%20me%20to%20the%20user%27s%20post%27s%20index%20page.,-User%20post%20index), Please add a test that redirects to the user's post's index page when you click on a **see all posts**.